docs/rules/no-ember-testing-in-module-scope.md

```diff
@@ -0,0 +1,35 @@
+## Ember.testing is not allowed in modules scope
+
+### Rule name: `no-ember-testing-in-module-scope`
+
+Since ember-cli-qunit@4.1.0 / ember-qunit@3.0.0, Ember.testing is only set to
```

Comment:
Can you wrap `Ember.testing` in backticks (for better highlighting) throughout this file?